python/taichi/misc/gui.py

```diff
@@ -342,7 +345,7 @@ def text(self, content, pos, font_size=15, color=0xFFFFFF):
         color = ti.core_vec(r, g, b, 1)
         self.canvas.text(content, pos, font_size, color)
 
-    def _make_field_base(gui, w, h, bound):
+    def _make_field_base(self, w, h, bound):
```

Comment:
Make this a `static_method` since `self` is not really used?

Reply:
```suggestion
    @staticmethod
    def _make_field_base(w, h, bound):
```